templates/minimal/zombienet.toml

```diff
@@ -0,0 +1,37 @@
+# Nodes might alternate in block production for a while, based on
```

Comment:
Might be out of date, pertaining to days when all nodes produced blocks.

Reply:
Made it more succinct here: [92dbdd8](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/92dbdd8b480e9381333b82beeec6e472dd3e7a2c)